ml-unit-test-client/src/test/ml-modules/root/test/suites/More Unit Tests/http-helpers.xqy

```diff
@@ -0,0 +1,5 @@
+import module namespace test="http://marklogic.com/roxy/test-helper" at "/test/test-helper.xqy";
+
+
+test:assert-http-get-status("v1/search?format=xml", $test:DEFAULT_HTTP_OPTIONS,200),
```

Comment:
I don't see a definition for $test:DEFAULT_HTTP_OPTIONS.

Reply:
Thanks for catching that. I was in the client directory when I committed and pushed, so it missed my modules changes. I pushed the modules changes now.